flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -69,6 +69,37 @@
  */
 class HistoryServerArchiveFetcher {
 
+	/**
+	 * Possible job archive operations in history-server.
+	 */
+	public enum JobArchiveOperation {
```

Comment:
```suggestion
	public enum ArchiveEventType {
```

I think type better describes this, because the other class is called `(Job)ArchiveEvent`.

Reply:
Yes, that is better name. Renamed, thanks.